### PR TITLE
testing/google-apputils: uprade to 0.4.2

### PR DESCRIPTION
**Pull request by little-dude**

### CODE REVIEW

File: testing/py-google-apputils/APKBUILD [hunk 1]
```diff
@@ -22,8 +22,9 @@ build() {
 package() {
 	cd "$builddir"
 	python setup.py install --prefix=/usr --root="$pkgdir" || return 1
+	chmod o+r "$pkgdir"/usr/lib/python2.7/site-packages/google_apputils-"$pkgver"-py2.7.egg-info/* || return 1
```
Comment (jirutka) [inline testing/py-google-apputils/APKBUILD]: Why is this needed?

Comment (little-dude) [inline testing/py-google-apputils/APKBUILD]: It's a workaround this issue: https://github.com/google/google-apputils/issues/4
Without it, the package is broken, see for instance: https://travis-ci.org/alpinelinux/aports/builds/134998764

I can do that in a seperate commit if you prefer.

Comment (jirutka) [inline testing/py-google-apputils/APKBUILD]: That’s okay, just please add a comment explaining that it’s a workaround for that issue and add link to the issue.

Comment (little-dude) [inline testing/py-google-apputils/APKBUILD]: :+1: 
I added the comment

**Comment by jirutka** [comment]
Merged in 0693cd94e453ec1bfb5b27c8fa229a9a40f41b69, thanks!

**Comment by little-dude** [comment]
Thank you!
